app/src/main/java/com/photoeditorsdk/android/app/KEditorDemoActivity.kt

```diff
@@ -78,6 +83,20 @@ class KEditorDemoActivity : Activity(), PermissionRequest.Response {
             // Set custom editor image export settings
             it.setOutputToGallery(Environment.DIRECTORY_DCIM)
             it.outputMode = OutputMode.EXPORT_IF_NECESSARY
+        }.also {
+            enableBrushPresets(it)
+        }
+
+    private fun enableBrushPresets(settingsList: PhotoEditorSettingsList) =
```

Comment:
Why are the brush presets in the DemoActivity of the demo app? Shouldn't they just be in the brush example app?